tests/cli/test_integration.py

```diff
@@ -82,7 +82,9 @@ def test_runpath_file(tmpdir, source_root):
         assert os.path.isfile("RUNPATH_WORKFLOW_1.OK")
 
 
-def test_ensemble_evaluator(tmpdir, source_root):
+def test_ensemble_evaluator(tmpdir, source_root, caplog):
+    import logging
+    caplog.set_level(logging.DEBUG, logger="ert_shared.ensemble_evaluator")
```

Comment:
I don't mind leaving the logging in, but if so we should move the import to the top of the file